region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -182,6 +215,79 @@
         "Emphasis on Careful.  There is no jump."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Leave With Mockball",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 16,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 1,
+            "openEnd": 1
+          },
+          "minExtraRunSpeed": "$1.2",
+          "maxExtraRunSpeed": "$3.9"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [1, 2],
+      "name": "Leave With Spring Ball Bounce",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithSpringBallBounce": {
+          "remoteRunway": {
+            "length": 16,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 1,
+            "openEnd": 1
+          },
+          "minExtraRunSpeed": "$1.2",
+          "maxExtraRunSpeed": "$3.9",
+          "movementType": "uncontrolled"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [1, 2],
+      "name": "Leave Space Jumping",
+      "requires": [],
+      "exitCondition": {
+        "leaveSpaceJumping": {
+          "remoteRunway": {
+            "length": 16,
+            "openEnd": 1
+          },
+          "minExtraRunSpeed": "$1.5",
+          "maxExtraRunSpeed": "$3.9"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [1, 2],
+      "name": "Leave With Temporary Blue",
+      "requires": [
+        {"getBlueSpeed": {
+          "usedTiles": 16,
+          "openEnd": 1
+        }},
+        "canChainTemporaryBlue"
```

Comment:
Can you jump over to the door?  Without testing I would guess its `can4HighMidAirMorph` to reach the door and `canTrickyJump` to blind stop on the runway.  And there'd be some jump in the water options.

Reply:
I had in mind that you jump with run speed into a morph + unmorph to make it onto the door frame ledge with temporary blue. It's intended to be reflected in the use of "getBlueSpeed" rather than "canShinecharge", since doing it this way we're not getting a shinecharge. Added a note now to clarify this